### ISSUE TITLE
Refactor `icons` from Tekton and Topology plugins to use MUI icons

### ISSUE THREAD

**Issue by divyanshiGupta** [janus-idp/backstage-plugins#716]
`icons` file is duplicated in Tekton and Topology plugins and in order to remove duplicate code we need to move this file to shared-package. Currently pf icons were being used for these plugins but to avoid adding a dependency on `patternfly/react-icons` in shared-package `icons` needs to be refactored to use similar MUI icons.

**Comment by BethGriggs** [comment]
I don't believe this issue is still valid with the intended migration away `shared-react`. If enhancements are still needed then an issue can be opened in Backstage Community Plugins where these plugins have migrated.
